src/DynamoCoreWpf/UI/Themes/Modern/DynamoModern.xaml

```diff
@@ -2482,6 +2482,42 @@
         </Setter>
     </Style>
 
+    <Style TargetType="Expander" x:Key="WorkspaceReferencesExpanderStyle">
```

Comment:
Hi @reddyashish I thought expander style has been added already, is this one different?

Reply:
The difference between this expander and the MenuExpanderStyle is the margin. There is more margin spacing for the elements in preferences panel.